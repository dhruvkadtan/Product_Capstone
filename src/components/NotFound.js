import React from "react";

export default class NotFound extends React.Component{
    render() {
        return(
            <div>
                <h3>Wrong URL</h3>
            </div>
        )
    }
}