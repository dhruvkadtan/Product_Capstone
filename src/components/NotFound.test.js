import React from 'react';
import renderer from 'react-test-renderer';
import { screen,render as renderRtl } from '@testing-library/react';
import  NotFound from './NotFound';

const setup = () => renderRtl(<NotFound/>);

describe('checking render',() => {
  test('render correctly', () => {
    const tree = renderer.create(<NotFound/>)
    expect(tree).toMatchSnapshot();
  })

})

describe('checking dom elements', () => {
  test('Correct heading', () => {
    setup();
    const heading = screen.getByText("Wrong URL")
    expect(heading).toBeInTheDocument();
})
})

