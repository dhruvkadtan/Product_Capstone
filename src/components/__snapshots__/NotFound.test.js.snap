// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checking render render correctly 1`] = `
<div>
  <h3>
    Wrong URL
  </h3>
</div>
`;
